src/frequenz/sdk/timeseries/_formula_engine/_formula_engine.py

```diff
@@ -174,11 +174,13 @@ async def apply(self) -> Sample:
     "HigherOrderFormulaBuilder3Phase",
 ]
 
+# pylint: disable=invalid-name
```

Comment:
Why is is an invalid name? Does it say? This looks strange. Please add a comment explaining both in the code and the commit message.

Reply:
Would be interesting to see the `pylint` message to see exactly what it is complaining about.